Tryouts/Plugins/ApplicationPlugins/datagrid/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@morganstanley/compose-example-grid",
```

Comment:
should be "@morgan-stanley/compose-example-grid"
Semantic versioning starts from 0.1.0